spec/spec.yaml

```
@@ -814,6 +814,16 @@ paths:
                 "code": "RESOURCE_CREATED",
                 "message": "A new cluster has been created with ID 'wqtlq'"
               }
+        "400":
+          description: Instance type not found
```

```suggestion
          description: Invalid request
```

Since the 400 response code is used for all sorts of invalid inputs, the headline should reflect that.